tests/rbd_mirror/rbd_mirror_utils.py

```diff
@@ -368,12 +390,35 @@ def mirror_status(self, *args):
 
     # Add Peer
     def peer_add(self, **kw):
-        return self.exec_cmd(
-            cmd="rbd mirror pool peer add {} {}".format(
-                kw.get("poolname"), kw.get("cluster_spec")
+            return self.exec_cmd(
+                    cmd="rbd mirror pool peer add {} {}".format(
+                        kw.get("poolname"), kw.get("cluster_spec")
+                    )
             )
+    
+    #Create bootstrap token on primary cluster
+    def bootstrap_peers(self, **kw):
+        return self.exec_cmd(
+                cmd="rbd mirror pool peer bootstrap create --site-name master {} > /root/bootstrap_token_master".format(
```

Comment:
In order to maintain consistently, we highly recommend using `f-string` formatters... i.e.
```
cmd=f"rbd mirror pool peer bootstrap create --site-name master {kw['poolname']} > /root/bootstrap_token_master"
```